### PR TITLE
Story6

### PR DESCRIPTION
**Pull request by zdrvk**

### CODE REVIEW

File: deletedGames/displayDeletedGames.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+function displayDeletedGames(array $games)
```
Comment (ashleycoles) [inline deletedGames/displayDeletedGames.php:3]: unit testtttttttssssssssss

File: src/Models/GameModel.php [hunk 2]
```diff
@@ -67,12 +67,17 @@ public function addNewGame(
...
     {
-
-        $query = $this->db->prepare("
+        if ($deleted == 1) {
+            $query = $this->db->prepare("
+        UPDATE `games` SET `games`.`deleted` = 0 WHERE `games`.`id` = :id;
+        ");
+        } else if ($deleted == 0) {
+            $query = $this->db->prepare("
         UPDATE `games` SET `games`.`deleted` = 1 WHERE `games`.`id` = :id;
```
Comment (ashleycoles) [inline src/Models/GameModel.php]: This bit you can make DRYer. You can have the query just once as a string, but pass in 1 or 0 depending on what $deleted is set to

File: src/displayAllGames.php [hunk 3]
```diff
@@ -4,7 +4,7 @@ function displayAllGames(array $games)
 {
     $title = '';
     foreach ($games as $game) {
-        if ($game->deleted !== 1) {
+        if ($game->deleted === 0) {
```
Comment (ashleycoles) [inline src/displayAllGames.php]: Rather than filtering out deleted/undeleted like this, it is actually more efficient to do with and SQL where condition.

**Review by ashleycoles — CHANGES_REQUESTED** [review]

File: src/displayAllGames.php (hunk 3 above)
Comment (zdrvk) [reply src/displayAllGames.php]: I might need some help with that one because I have no idea where do do the query

**Pull request merged**